flujo/domain/evaluation.py

```
@@ -11,3 +20,252 @@ class EvaluationScore(BaseModel):
+def _ensure_checklist_items(checklist: Checklist) -> Checklist:
+    if checklist.items is None:
+        checklist.items = []
+    cleaned: list[ChecklistItem] = []
+    for item in checklist.items:
+        if isinstance(item, ChecklistItem):
+            cleaned.append(item)
+        elif isinstance(item, dict):
+            try:
+                cleaned.append(ChecklistItem.model_validate(item))
+            except Exception:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)